src/backend/_scss/application.scss

```diff
@@ -416,7 +416,9 @@ a {
   box-shadow: 0 1px 2px $event-box-shadow;
   cursor: pointer;
   display: block;
-  margin: 0 12px 12px 0;
+  margin: 12px;
```

Comment:
Why this is overwritten by the three lines @mbad0la , I was considering shorthand like these :)